CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Extended, updated, and corrected documentation for filter functions [#118](https://github.com/matchms/matchms/pull/118)
+- Add basic submodule documentation and more code examples [#128](https://github.com/matchms/matchms/pull/128)
```

Comment:
Remove `add` or move to `### Added` section.

Reply:
Moved it to ``### Added``.